### PR TITLE
New article: Private network access update

### PR DESCRIPTION
**Pull request by agektmr**
A new article about "Private Network Access update".

cc: @mihajlija

### CODE REVIEW

File: site/en/blog/private-network-access-update/index.md [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+API](https://bugzilla.mozilla.org/show_bug.cgi?id=1620573). You may want to use
+both APIs during the transition.
+
+{% endAside %}
+
+### Avoid crossing address spaces
+
+The simplest way around the issues caused by private network requests is to
+avoid straddling the boundary between public and private IP addresses. This can
+be achieved in one of two ways:
```
Comment (mihajlija) [inline site/en/blog/private-network-access-update/index.md:185]:
```suggestion
avoid straddling the boundary between public and private IP addresses. There are
two ways to achieve this.
```

File: site/en/blog/private-network-access-update/index.md [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+* [https://report-uri.com](https://report-uri.com)
+* [https://uriports.com](https://uriports.com)
+
+{% Aside %}
+
+[Please let us know](https://github.com/GoogleChrome/web.dev/issues) if you know
+any other options so we can add them to the list.
+
+We are not aware of any open source solutions that can accumulate reports in a
+database. We welcome to list them here as well if you ever build one.
```
Comment (mihajlija) [inline site/en/blog/private-network-access-update/index.md:158]:
```suggestion
We are not aware of any open source solutions that can accumulate reports in a
database. If you know any other solutions, especially open source ones, [let us know](https://github.com/GoogleChrome/web.dev/issues) so we can add them to the list.
```

File: site/en/blog/private-network-access-update/index.md [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+asked to provide an explicit grant to the initiator. The request is only sent if
+the grant is successful.
+
+In short, a CORS preflight request is an HTTP `OPTIONS` request carrying some
+`Access-Control-Request-*` headers indicating the nature of the subsequent
+request. The server can then decide whether or not to grant fine-grained access
+by responding `200 OK` with `Access-Control-Allow-*` headers.
+
+More details on this can be found in [the
+specification](https://wicg.github.io/private-network-access).
```
Comment (mihajlija) [inline site/en/blog/private-network-access-update/index.md:241]:
```suggestion
Find more details about this in the
specification](https://wicg.github.io/private-network-access).
```

**Review by mihajlija — APPROVED** [review]

File: site/en/blog/private-network-access-update/index.md [hunk 4]
```diff
@@ -0,0 +1,256 @@
+---
+layout: 'layouts/blog-post.njk'
+title: Private Network Access (CORS-RFC1918) updates
+authors:
+  - agektmr
+  - titouan
+description: TBD
```
Comment (mihajlija) [inline site/en/blog/private-network-access-update/index.md:7]: Please add the description here.

File: site/en/blog/private-network-access-update/index.md [hunk 5]
```diff
@@ -0,0 +1,256 @@
...
+being allowed to send arbitrary requests.
+
+**Private network requests** are requests whose target server's IP address is
+more private than that from which the request initiator was fetched. For
+example, a request from a public website (`https://example.com`) to a private
+website (`http://router.local`), or a request from a private website to
+localhost.
+
+{% Img src="image/YLflGBAPWecgtKJLqCJHSzHqe2J2/nSGfU9xMJxIy6lMjJiMx.png",
+alt="ALT_TEXT_HERE", width="800", height="512" %}
```
Comment (mihajlija) [inline site/en/blog/private-network-access-update/index.md:54]: Missing alt description here.

**Comment by agektmr** [comment]
This pull request already got LGTM from @mihajlija but looks like it needs another from one of content owners.
@kaycebasques @jpmedley can you help?

**Comment by agektmr** [comment]
This is ready to be merged any time.

**Review by robdodson — APPROVED** [review]

**Review by mihajlija — APPROVED** [review]

**Pull request merged**